src/runtime_src/core/common/info_aie.cpp

```diff
@@ -277,10 +494,11 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
     pt.put("error_msg", ex.what());
     return pt;
   }
-
   try {
     // On Pcie platforms use driver calls to get AIE Shim info
-    pt_shim = asd_parser::get_formated_tiles_info(device, asd_parser::aie_tile_type::shim);
+
+    if (pt_shim.empty()) 
```

Comment:
This check was added because the pt_shim for edge device will get overwritten with this call for pcie platforms. I have now modified it further to check for pcie platform in the catch of "not edge device".